core/domain/suggestion_services.py

```diff
@@ -1826,3 +1828,696 @@ def update_question_suggestion(
     _update_suggestion(suggestion)
 
     return suggestion
+
+
+def _create_translation_review_stats_from_models(
```

Comment:
```suggestion
def _create_translation_review_stats_from_model(
```

Should be singular since it only accepts one model. Here and below.

Reply:
Done